src/main/java/de/tum/in/www1/artemis/service/connectors/VersionControlService.java

```diff
@@ -12,7 +12,13 @@
 
 public interface VersionControlService {
 
-    void configureRepository(URL repositoryUrl, Set<User> users);
+    /**
+     * Configure the repository for the student(s), this mainly includes
```

Comment:
I believe some parts of the documentation got lost here

Reply:
thanks, I fixed it